mslib/msui/_tests/test_editor.py

```diff
@@ -28,7 +28,7 @@
 import mock
```

Comment:
I do miss the general comment field as on bitbucket

please also look into the mss_qt.py for unused imports, see pylint or pycharm